Y2F/@optimizerFORCES/compileSolverInterfaceCode.m

```diff
@@ -160,11 +160,12 @@ function compileSolverInterfaceCode( self )
                     '-ldecimal', '-lirc', '-lmmt', '-lsvml_dispmt', ...
                     legacyLibs, '-lIPHLPAPI.lib', '-largeArrayDims', '-silent');
             else
+                iphlpapi_path = ['-L"', matlabroot, '\sys\lcc64\lcc64\lib64"'];
```

Comment:
Are you sure this works? LCC is not MinGW.

Reply:
yes, the library is compatible with MinGW. MinGW does not always come with the iphlpapi automatically in its path so it's a portable way of linking it properly